### PR TITLE
Added issue/pr template.

### PR DESCRIPTION
**Pull request by gyliu513**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/270)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+**What did you expect to happen:**
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- Fairing version:
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- Kubeflow version:
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

**Comment by karthikv2k** [comment]
Thanks @gyliu513 !

**Pull request merged**